### ISSUE TITLE
add CERES4.1 EBAF

### ISSUE THREAD

**Issue by zyuying** [PCMDI/PCMDIobs-cmor-tables#26]

**Comment by zyuying** [comment]
@gleckler1  OK got it

**Comment by zyuying** [comment]
@gleckler1 , The CERES EBAF 4.1 TOA and surface products are not available for download yet.
( https://ceres.larc.nasa.gov/products-info.php?product=EBAF-TOA )

**Comment by zyuying** [comment]
@gleckler1 , I've put the CERES EBAF 4.1 TOA data in /p/user_pub/pmp/pmp_obs_preparation/orig/data/CERES_EBAF4.1

The CERES EBAF 4.1 has been added to PMPObs_source_id.json, and I've generated CERES4.1-input.json and runCmor_CERES4.1_2D.py. These files has been merged to the master code. 

When I run the code, it shows
! Error: Table  is defined for cmor_version 3.500000, this library version is: 3.3.2, 3.300000
I think I need update my CMOR version, since I still use CMOR3.3.3.
